source/js/components/share-button-group/share-button-group.scss

```diff
@@ -1,13 +1,12 @@
 .share-button-group {
```

Comment:
`marign-right` didn't get added to the right elements. Buttons weren't always in the same size as a result.